.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: ["main"]
+    branches: [ "main" ]
```

Comment:
Could we avoid the unrelated formatting changes? I think this might be your editor, is that right?

Reply:
It was reformated by my IDE. Those changes were dropped.